src/plugins/image_embeddable/public/image_embeddable/image_embeddable.tsx

```diff
@@ -48,6 +54,10 @@ export class ImageEmbeddable extends Embeddable<ImageEmbeddableInput> {
   private ImageEmbeddableViewer = (props: { embeddable: ImageEmbeddable }) => {
     const input = useObservable(props.embeddable.getInput$(), props.embeddable.getInput());
 
+    React.useLayoutEffect(() => {
+      import('./image_embeddable_lazy');
```

Comment:
Importing `scss` directly added 5kb to the main bundle because of all the scss runtime helpers